src/test/java/com/google/api/codegen/testdata/php_main_library.baseline

```diff
@@ -230,6 +228,7 @@ class LibraryServiceApi
         return self::$archivedBookNameTemplate;
     }
 
+
```

Comment:
Oops. I have too many tabs open.

That sounds like a bug in the snippet language, since the whole point of it is to get formatting right. I strongly prefer to avoid extra blank lines unless it's a thing that a particular language community likes to do.